cypress/integration/home_page_spec.js

```diff
@@ -40,13 +62,15 @@ describe('Home page user flow', () => {
 
     it('Should be able to search Pokemon by Name', () => {
         cy.get('input[type="search"]')
-          .type('CharmAnder')
-          .should('have.value',  'CharmAnder')
+          .type('Pikachu')
+          .should('have.value', 'Pikachu')
           .get('button')
           .click()
-          .get('.pokemon-card')
-          .contains('charmander')
-          .get('img[src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/4.png"]')
+          .get('.pokemon-details-page')
+          .contains('25')
+          .get('.pokemon-details-header')
+          .contains('pikachu')
+          .get('img[src="https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/25.png"]')
           .should('be.visible')
     });
```

Comment:
In order to make sure we test everything, maybe we can add a test that is looking for all of the category headers as well? Just to make sure that the info is in fact there. EDIT: I see you actually did this below! Awesome job!